protocol/avdl/chat1/common.avdl

```diff
@@ -187,6 +187,12 @@ protocol common {
     bytes k; // Verifying key
   }
 
+  record SignEncryptedData {
```

Comment:
Yeah. It's got different guarantees (signed) and uses different functions to open.